packages/fxa-auth-server/package.json

```diff
@@ -113,7 +113,6 @@
     "moment-timezone": "^0.5.35",
     "mozlog": "^3.0.2",
     "mysql": "^2.18.1",
-    "node-uap": "https://github.com/mozilla-fxa/node-uap.git#494714db3e65a4dbfae772f536c8697269174574",
```

Comment:
Do you know why this was pinned?

Reply:
I don't know the whole history.  It was pinned to Danny's fork.  Then Dan's.  Then it moved to mozilla-fxa.  node-uap also deleted some commits on the original repo so maybe that was a motivation to keep a fork?